src/azure-account.ts

```diff
@@ -386,8 +386,14 @@ export class AzureLoginHelper {
 						placeHolder: 'https://management.local.azurestack.external',
 						ignoreFocusOut: true
 					});
+					const tenantId = await window.showInputBox({
```

Comment:
where will this input box be shown?, if we do it for TenantId, why not for ARm endpoint as well?